app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,10 @@
+class AdminController < ApplicationController
+  before_action :authorized?
+  private
```

Comment:
add empty lines before and after

Reply:
install https://github.com/rubocop-hq/rubocop and use it

Reply:
installed and checked.